scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,247 @@
+            sh:datatype xsd:string ;
+            sh:description " Use of the functionalities from the standard. E.g. which Actions are used."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of usedStandardFunctions failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "usedStandardFunctions"@en ;
+            sh:order 3 ;
+            sh:path scenario:usedStandardFunctions ] ;
+    sh:targetClass scenario:General .
+
+scenario:ParticipantsShape a sh:NodeShape ;
+    sh:property [ skos:example "2" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic temporary objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of temporaryTrafficObjects failed!"@en ;
+            sh:name "temporaryTrafficObjects"@en ;
+            sh:order 1 ;
+            sh:path scenario:temporaryTrafficObjects ],
+        [ skos:example "10" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of numberTrafficObjects failed!"@en ;
+            sh:name "numberTrafficObjects"@en ;
+            sh:order 0 ;
+            sh:path scenario:numberTrafficObjects ],
+        [ skos:example "myDriver, BestPedestrianController2000" ;
+            sh:datatype xsd:string ;
+            sh:description "Usage of simulation Core internal or external controllers."@en ;
+            sh:message "Validation of controllers failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "controllers"@en ;
+            sh:order 5 ;
+            sh:path scenario:controllers ],
+        [ skos:example "8" ;
+            sh:datatype xsd:unsignedInt ;
+            sh:description "Count of traffic permanent objects"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of permanentTrafficObjects failed!"@en ;
+            sh:name "permanentTrafficObjects"@en ;
+            sh:order 2 ;
+            sh:path scenario:permanentTrafficObjects ],
+        [ skos:example "via trajectory, via support points" ;
+            sh:datatype xsd:string ;
+            sh:description "Type of movement used by traffic participants according to the OpenSCENARIO standard."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of movementDescription failed!"@en ;
+            sh:name "movementDescription"@en ;
+            sh:order 4 ;
+            sh:path scenario:movementDescription ],
+        [ skos:example "stopIfTTCLowerThan" ;
+            sh:datatype xsd:string ;
+            sh:description "Scenario/domain specific commands or actions"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of customCommands failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "customCommands"@en ;
+            sh:order 6 ;
+            sh:path scenario:customCommands ] ;
+    sh:targetClass scenario:Participants .
```

Comment:
"Participants" seems like a unwise naming as we already have participants as entity participating in a data ecosystems. 

Would a renaming into "Objects" make sense?

Reply:
Yes this could lead to some confusion. However, `Objects` would refer also in the context of a scenario to anything also static things like a tree or so. I'd like to stick somehow to something with participants, would `trafficParticipants` would be alright for you?

Reply:
@MircoNierenz Fin already adjusted the shacl